src/DynamoCoreWpf/UI/Themes/Modern/DynamoConverters.xaml

```diff
@@ -63,6 +63,7 @@
     <controls:PackageSearchStateToStringConverter x:Key="PackageSearchStateToStringConverter" />
     <controls:EmptyStringToCollapsedConverter x:Key="EmptyStringToCollapsedConverter" />
     <controls:NonEmptyStringToCollapsedConverter x:Key="NonEmptyStringToCollapsedConverter" />
+    <controls:InstalledButtonTextConverter x:Key="InstalledButtonTextConverter" />
```

Comment:
How about the converters around dates, do we need to add them here?

Reply:
As in the PrettyDateConverter? That one existed already, I'm seeing it on line 85.